herbert_core/herbert_init.m

```diff
@@ -127,11 +127,11 @@ function application_off(app_name)
         else
             disp(['Function ',app_name,'_off.m not found in ',rootpath])
             disp('Clearing rootpath and subdirectories from Matlab path in any case')
+            paths = genpath(rootpath);
+            warn_state=warning('off','all');    % turn of warnings (so don't get errors if remove non-existent paths)
+            rmpath(paths);
+            warning(warn_state);    % return warnings to initial state            
```

Comment:
I think we should still have the logic preventing paths in `MATLABPATH` being removed

Reply:
in this form -- yes

Reply:
I do not actually understand why it is removing the MATPABPATH if you have this variable outside of the package. 
You can add this logic to the removal code. 

but in the form I committed, the second removal would not occur so should be irrelevant to MATLABPATH issue